### PR TITLE
chore: sort imports

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
Sort imports in ts/tsx files. This matches what we do in the backend and web client.

### CODE REVIEW

File: dev-client/.prettierrc [hunk 1]
```diff
@@ -1,7 +1,25 @@
 {
   "arrowParens": "avoid",
+  "singleQuote": true,
+  "semi": true,
```
Comment (knipec) [inline dev-client/.prettierrc:4]: Questions: This means "anyplace that could have a semicolon should have a semicolon", right?

File: dev-client/.depcheckrc
Comment (knipec) [inline dev-client/.depcheckrc:1]: Question: Were the spacing changes here just done manually, or is there a prettier rule that did this?

File: dev-client/.prettierrc (hunk 1 above)
Comment (paulschreiber) [reply dev-client/.prettierrc:4]: Yes. We're already doing that. I added it to the file so the prettier configuration better matches the web client.

File: dev-client/.depcheckrc
Comment (paulschreiber) [reply dev-client/.depcheckrc:1]: I did manually sort the array.

**Review by knipec — APPROVED** [review]
I'd just check with @tm-ruxandra before submitting that she won't mind the thrash of this change, since she didn't say either way on the slack thread

**Pull request merged**